crates/iceberg/src/spec/manifest.rs

```diff
@@ -1237,7 +1237,7 @@ pub fn write_data_files_to_avro<W: Write>(
 }
 
 /// Parse data files from avro bytes.
-pub fn parse_data_file_from_avro<R: Read>(
+pub fn read_data_file_from_avro<R: Read>(
```

Comment:
Hi, it returns `Vec<DataFile>` instead of `DataFile`, so `read_data_file_from_avro` is an incorrect API name. It would be better to rename it to `read_data_files_from_avro`.

Alternatively, we could rename `write_data_files_to_avro` to `write_data_file_to_avro` to eliminate the concept of plurals from our API.

Reply:
Nice catch! I have rename it.